CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 2.0.0-nullsafety.0
+## 2.0.0
```

Comment:
We have been keeping the `-nullsafety.0` changelog entries in place and adding new ones for the stable release.

Should we be changing our approach? cc @jakemac53

Reply:
I don't know... I can see an argument either way.

In the case of breaking changes, they can be a bit buried with the current approach.